applications/nrf_desktop/doc/battery_charger.rst

```diff
@@ -0,0 +1,53 @@
+.. _battery_charger:
+
+Battery charger module
+######################
+
+The ``battery_charger`` module is responsible for battery charging control.
+
+Module Events
+*************
+
++------------------+---------------------+---------------------+-------------------------+------------------+
+| Source Module    | Input Event         | This Module         | Output Event            | Sink Module      |
++==================+=====================+=====================+=========================+==================+
+|                  |                     | ``battery_charger`` | ``battery_state_event`` | :ref:`led_state` |
++------------------+---------------------+                     +-------------------------+------------------+
+| :ref:`usb_state` | ``usb_state_event`` |                     |                         |                  |
++------------------+---------------------+---------------------+-------------------------+------------------+
+
+Configuration
+*************
+
+The module implemented in ``battery_charger.c`` uses the Zephyr :ref:`zephyr:gpio` driver to control and monitor battery charging. For this reason, you should set :option:`CONFIG_GPIO` option.
+
+Set option ``CONFIG_DESKTOP_BATTERY_CHARGER_DISCRETE`` to enable the module (by default ``CONFIG_DESKTOP_BATTERY_CHARGER_NONE`` is selected - module is disabled).
+
+Following module configuration options are available:
+* ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PIN`` - pin to which CSO (charging status) output from charger is connected
+* CSO pin pull:
+  * ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PULL_NONE`` - disabled (default option)
```

Comment:
```suggestion
  * ``CONFIG_DESKTOP_BATTERY_CHARGER_CSO_PULL_NONE`` - disabled (default setting)
```

Lines 29-31: what is disabled, on and off?

Reply:
CSO pin pull, mentioned above